DFC.FutureAccessModel.AreaRouting.UnitTests/Providers/ExpressionActionProviderFixture.cs

```diff
@@ -160,7 +160,7 @@ public void GetOutwardCodeFromMeetsExpectation(string theCandidate, string theEx
             var sut = MakeSUT();
```

Comment:
looks like this line of code is redundant

Reply:
I have removed redundant code.